strr-platform-web/app/pages/auth/login.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+const { t, locale } = useI18n()
+const keycloak = useKeycloak()
+const redirectUrl = useRuntimeConfig().public.baseUrl + locale.value + '/platform/dashboard'
+const logoutUrl = useRuntimeConfig().public.baseUrl + locale.value + '/auth/login'
+
+keycloak.setLogoutRedirectUrl(logoutUrl)
+
+// page stuff
+useHead({
+  title: t('platform.title.login')
+})
+
+setBreadcrumbs([
+  { label: t('label.bcregDash'), to: useRuntimeConfig().public.registryHomeURL + 'dashboard' }
+])
```

Comment:
do you think this should have a 'login' crumb too?

Reply:
Probably a good idea, should the first breadcrumb link go to the registry dashboard still? or housing? I could add a todo comment instead for now